src/gallery.md

```diff
@@ -174,6 +174,12 @@ This page collects screenshots from various simulations based on MFEM. Additiona
 
 *[Triple point shock interaction](https://computation.llnl.gov/projects/blast/triple-point-shock-interaction) on 4 elements of order 12.  Note the element curvature and the high variation of the field inside the lower right element.*
 
+---
+
+[![](img/gallery/nstxu_hhfw.png)](img/gallery/nstxu_hhfw.png)
```

Comment:
Do you happen to have a higher resolution image by any chance?

Reply:
Let me generate another image, when the cluster comes back ;D

Reply:
Added a new figure. Please take a look

Reply:
That looks very sharp!